.mergify.yml

```diff
@@ -10,5 +10,7 @@ pull_request_rules:
         method: rebase
         rebase_fallback: merge
         strict: smart
-      dismiss_reviews: {}
+      dismiss_reviews:
```

Comment:
Should this be a separate rule name and set of actions. I somehow cannot understand a `condition` that states check for > 2 approving issues having an action to remove stale reviews. IOW, if the conditions are not met the actions won't be performed, whereas we want to remove any approving reviews if a PR is updated (not only when it is ready for merge).

Reply:
A 2nd rule has been added